From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: VeridianOS <veridian@veridian-os.org>
Date: Mon, 17 Feb 2026 00:00:00 +0000
Subject: [PATCH 2/2] Recognize veridian as a valid OS in GCC's config.sub

GCC ships its own copy of config.sub (at the top level of the source
tree).  This patch mirrors the change made to binutils' config.sub so
that GCC's configure scripts can parse x86_64-veridian,
aarch64-veridian, and riscv64-veridian target triples.

The entry is added at the end of the long list of recognized OS names,
just before the closing parenthesis.
---
 config.sub | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/config.sub
+++ b/config.sub
@@ -1749,7 +1749,8 @@
 	     | onefs* | tirtos* | phoenix* | fuchsia* | redox* | bme* \
 	     | midnightbsd* | amdhsa* | unleashed* | emscripten* | wasi* \
 	     | nsk* | powerunix* | genode* | zvmoe* | qnx* | emx* | zephyr* \
-	     | fiwix* )
+	     | fiwix* \
+	     | veridian* )
 		;;
 	# This one is extra strict with allowed versions
 	sco3.2v2 | sco3.2v[4-9]* | sco5v6*)
--
2.47.1

